plat/rockchip/common/plat_pm.c

```diff
@@ -154,14 +143,22 @@ int rockchip_pwr_domain_on(u_register_t mpidr)
  ******************************************************************************/
 void rockchip_pwr_domain_off(const psci_power_state_t *target_state)
 {
-	assert(RK_CORE_PWR_STATE(target_state) == PLAT_MAX_OFF_STATE);
+	if (RK_CORE_PWR_STATE(target_state) != PLAT_MAX_OFF_STATE)
+		return;
```

Comment:
It's a bit suspicious that you've changed an assert to a silent return. Was this condition being fired in certain circumstances or are you just being defensive? In general, asserts are preferred if this really is a programming error.